fe/fe-core/src/main/java/com/starrocks/lake/ShardDeleter.java

```diff
@@ -19,57 +22,65 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.stream.Collectors;
 
 public class ShardDeleter extends LeaderDaemon {
     private static final Logger LOG = LogManager.getLogger(ShardDeleter.class);
 
-    @SerializedName(value = "shardIds")
-    private final Set<Long> shardIds;
+    private List<Long> getAllPartitionId() {
```

Comment:
it returns shard group id set, not partition id set, need change the member method name.